docs/source/algorithms/Stitch1DMany-v1.rst

```diff
@@ -40,6 +40,9 @@ We can specify manual scale factors to use by setting
 of scale factors from the selected period. These scale factors are then applied
 to all other periods when stitching.
 
+The workspace that provides the scale for stitching output can be chosen by
+specifying the desired index through :literal:`IndexOfReference` property.
```

Comment:
Maybe clarify: The reference workspace will not be scaled; the other will be scaled to match the reference.